lua/packer.lua

```diff
@@ -784,6 +790,22 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+--- Completion for listing snapshots in `snapshot_path`
+--- Intended to provide completion for PackerRollback command
+--- TODO: using vim.fn.readdir to get entries the snapshot directory
+packer.snapshot_complete = function(lead, _, _)
+  local completion_list = {}
+  local res = io.popen('ls ' .. config.snapshot_path, 'r')
+  for entry in res:lines() do
+    if vim.startswith(entry, lead) then
+      table.insert(completion_list, entry)
+    end
+  end
+  res:close()
+  table.sort(completion_list)
```

Comment:
Is the sort necessary? Since we're doing prefix matching anyway, I'd expect it doesn't really matter if we sort.